static/app/views/auth/registerForm.tsx

```diff
@@ -21,71 +38,87 @@ function RegisterForm({authConfig}: Props) {
 
   const [error, setError] = useState('');
 
+  const signUpProviderUrls = getSignUpProviderUrls();
+
+  const orgSlug = ConfigStore.getState().customerDomain?.subdomain;
+  // TODO(Telemetry): Remove the condition below after checking that everything is working fine
+  const displayExternalProviders = orgSlug === 'pri-sentry';
```

Comment:
Since the invited user might not be logged in yet, we cannot use our feature flag tool in this case. To test it before making it public, I’m adding checking against an org that I use specifically for testing purposes